airbyte-integrations/connectors/destination-jdbc/src/main/resources/spec.json

```diff
@@ -1,7 +1,6 @@
 {
   "documentationUrl": "https://docs.airbyte.io/integrations/destinations/postgres",
   "supportsIncremental": true,
-  "supported_destination_sync_modes": ["overwrite", "append", "append_dedup"],
```

Comment:
just remove `append_dedup` and keep the others

Reply:
makes sense.